.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,12 +1,45 @@
 ---
 name: Feature Request
-about: You want to suggest a new feature for Atmosphère.
-
+about: Suggest a new feature for Atmosphère.
+#assignees:
 ---
 
 ## Feature Request
-
+[ If any section does not apply, use nil. ]  
+[ Lines between [ ] (square brackets) should be removed before posting. ]
 ### What feature are you suggesting?
+#### Overview:
+- [ Include the basic, high-level concepts for this feature here. ]
 
-### Why would this feature be useful?
+#### Smaller Details: 
+- [ These may include specific methods of implementation etc. ]
+
+#### Nature of Request:  
+[ Remove all that do not apply to your request. ]
+- Addition
+  - [ Ex: Addition of certain original features or features from other community projects. ]
+  - [ If you are suggesting porting features or including features from other projects, include what license they are distributed under and what, if any libraries those project use. ]
+- Change
+- Removal
+  - [Ex: Removal of certain features or implementation due to a specific issue/bug or because of low quality code, etc.]
```

Comment:
I really don't think we need three separate sections for describing a feature request. Just a "feature description" section would be enough.

Reply:
The reasoning behind the 3 sections is too weed out generic and low effort requests. I do agree it could be condensed but this section was made based upon my experiences with other repositories and contributions.